styleguide/source/assets/scss/01-atoms/_skip-to-main-content.scss

```diff
@@ -0,0 +1,12 @@
+//Accesibility skip to main
+//https://accessibility.oit.ncsu.edu/it-accessibility-at-nc-state/developers/accessibility-handbook/mouse-and-keyboard-events/skip-to-main-content/
+.ama__header__skip-nav {
```

Comment:
Since the skip nav is it's own atom, it seems safe to assume it would be it's own block level element. I suggest updating the class to `.ama__skip-nav`